docs/en/GettingStarted.md

```diff
@@ -7,10 +7,11 @@ permalink: docs/en/getting-started.html
 next: using-matchers
 ---
 
-Install Jest using `npm`:
+Install Jest using `npm` or [`yarn`](https://yarnpkg.com/en/package/jest):
 
 ```
 npm install --save-dev jest
+yarn add --dev jest
 ```
```

Comment:
I have a feeling that inexperienced users may try to do both, as the commands are in the same block. How about:

Install Jest using `npm`:	
  		  
```
npm install --save-dev jest
```

or `yarn`:

```
yarn add --dev jest
```